discohash.cpp

```diff
@@ -1,140 +1,140 @@
+// also known as BEBB4185
 // Copyright 2020 Cris Stringfellow
 // https://github.com/dosyago/discohash
 #include <cstdio>
 #include <inttypes.h>
 #include <cstring>
+#include <iostream>
 #include "discohash.h"
 
 constexpr int STATE = 32;
-constexpr int STATE64 = STATE >> 3;
 constexpr int STATEM = STATE-1;
-constexpr int HSTATE64M = (STATE64 >> 1)-1;
+constexpr int STATE64 = STATE >> 3;
+constexpr int HSTATE64 = STATE64 >> 1;
 constexpr int STATE64M = STATE64-1;
+constexpr int HSTATE64M = HSTATE64-1;
 alignas(uint64_t) uint8_t disco_buf[STATE] = {0};
 constexpr uint64_t P = 0xFFFFFFFFFFFFFFFF - 58;
 constexpr uint64_t Q = 13166748625691186689U;
```

Comment:
UINT64_C(13166748625691186689) would be better for windows

Reply:
OK!